src/framework/components/element/component.js

```diff
@@ -1027,14 +1132,21 @@ class ElementComponent extends Component {
     }
 
     /**
-     * The minimum size that the font can scale to when autoFitWidth or autoFitHeight are true.
+     * Sets the minimum size that the font can scale to when `autoFitWidth` or `autoFitHeight` are
```

Comment:
good place to use a link {@link autoFitWidth} or {@link autoFitWidth | Auto Fit Width}

Reply:
Done! I have added lots of other links to this module too.